src/main/java/seedu/address/model/hotel/bill/Cost.java

```diff
@@ -21,7 +21,7 @@
      */
     public Cost(String cost) {
         requireNonNull(cost);
-        checkArgument(isValidCost(cost), MESSAGE_CONSTRAINTS);
+        checkArgument(isValidCost(cost), cost);
```

Comment:
The second parameters should be the error message

Reply:
it seems that you haven't resolved this.

Reply:
Oh this is outdated, I've resolved it when i fixed the bug for setrcost!